src/Symfony/Component/HttpFoundation/BinaryFileResponse.php

```diff
@@ -350,7 +350,7 @@ public static function trustXSendfileTypeHeader()
      *
      * @return BinaryFileResponse
      */
-    public function deleteFileAfterSend($shouldDelete)
+    public function deleteFileAfterSend($shouldDelete = true)
```

Comment:
IMHO this method (named as is) should not take any argument and just enable the feature (giving the possibility to disable it with a different method), or be named `setDeleteFileAfterSend()`.

Btw regarding the doc, it's said this behavior can be enabled, not that is the default one